reconcile/utils/jump_host.py

```diff
@@ -41,13 +41,29 @@ def cleanup(self):
 
 class JumpHostSSH(JumpHostBase):
     bastion_tunnel = {}
+    local_ports = []
     tunnel_lock = threading.Lock()
 
     def __init__(self, jh, settings=None):
         JumpHostBase.__init__(self, jh, settings=settings)
 
         self.known_hosts = self.get_known_hosts(jh)
         self.init_known_hosts_file()
+        self.local_port = self.get_random_port() \
+            if jh['localPort'] is None else jh['localPort']
+        self.remote_port = jh['remotePort']
+
+    def get_local_port(self):
+        return self.local_port
+
+    @staticmethod
+    def get_random_port():
```

Comment:
Just for clarity, suggest get_unique_random_port

Purely cosmetic though.  If it causes pain or a bunch of retesting it's not worth it